update rule 4 to remove the mention of #serious-off-topics (#87)
TCCPP/wiki: bot-articles/meta/rules.md

1. Use common sense. We do not tolerate slurs, bigotry of any kind (racism/sexism/homophobia/transphobia etc.), or glorification of violence or death.
2. Don't randomly ping / DM users. Just post your question in one of the help channels.
3. No advertising other Discord servers.

> 4. Keep the content SFW (safe-for-work), non-controversial and non-sensitive. This means no discussion of politics, ethics, current affairs and similar topics. This is also not the place to discuss your mental health problems.

Reviewer: ```suggestion
4. Keep the content SFW (safe-for-work). This server is not the place to debate controversial or sensitive topics, or political issues. This server is also not the place to discuss your mental health problems.
```
